test/living-html/htmlelement.js

```diff
@@ -0,0 +1,27 @@
+﻿"use strict";
+
+var jsdom = require("../..");
+
+exports["unknown elements should return HTMLUnknownElement"] = function (t) {
```

Comment:
Huh, not sure if that'll work with our current structure. Although I guess we could a put a list of all these special cases in an array or so. Are these documented explicitly somewhere?